src/utils/http-error-message.ts

```diff
@@ -13,7 +13,7 @@ export enum ErrorMessage {
     LISTALL_EW_404 = "The requested resource Tier is not found.",
     UPDATE_EW_400 = "Cloning an EdgeWorkers id is required in order to edit the Resource Tier id.",
     AUTH_TOKEN_ERROR = "ERROR: Unable to create authentication token.",
-    AUTH_TOKEN_404 = "Secret key could not be found.",
+    AUTH_TOKEN_404 = "Property could not be found for the specified hostname. Please specify a valid host name.",
```

Comment:
extra space between hostname

Reply:
You should keep this error msg "Secret key could not be found."
Pulsar API can return this error when the property was found but it didn't have secret key.

Reply:
I have removed this. I m going to be passing along the error message received from pulsar.